src/uwuw/tests/uwuw_unit_tests.cpp

```diff
@@ -148,11 +148,11 @@ TEST_F(UWUWTest, mat_write) {
   std::string openmc_rep = mat.openmc();
   std::cout << openmc_rep << std::endl;
   std::stringstream expected_rep;
-  expected_rep << "<material id=\"1\" name=\"Water\" >\n";
-  expected_rep << "  <density value=\"1.\" units=\"g/cc\" />\n";
-  expected_rep << "  <nuclide name=\"H1\" wo=\"6.6667e-01\" />\n";
-  expected_rep << "  <nuclide name=\"O16\" wo=\"3.3333e-01\" />\n";
-  expected_rep << "</material>\n";
+  expected_rep << "  <material id=\"1\" name=\"Water\" >\n";
```

Comment:
Thank you for taking care of this! Saw this was why the tests were failing and meant to send a PR your way but you beat me to it.